Makefile

```diff
@@ -10,6 +10,7 @@ WARNINGS=RUSTDOCFLAGS="-D warnings"
 DEBUG_ASSERTIONS=RUSTFLAGS="-C debug-assertions"
 ALL_FEATURES_BUT_ASYNC=--features concurrent,testing
 BUILD_KERNEL_ERRORS=BUILD_KERNEL_ERRORS=1
+BUILD_PROVING_SERVICE_PROTOS=CAN_WRITE_TO_SRC=1
```

Comment:
Should we just have a single environment variable that we use for all crates? That is, instead of relying on `BUILD_KERNEL_ERRORS`, `BUILD_PROVING_SERVICE_PROTOS`, and `DOCS_RS` we just have `CAN_WRITE_TO_SRC` and use it for all crates in this repo?